custom_components/localtuya/light.py

```diff
@@ -97,8 +154,21 @@ def __init__(
             MIRED_TO_KELVIN_CONST
             / self._config.get(CONF_COLOR_TEMP_MAX_KELVIN, DEFAULT_MAX_KELVIN)
         )
-        self._is_white_mode = True  # Hopefully sane default
         self._hs = None
+        self._effect = None
+        self._effect_list = []
+        self._scenes = None
+        if self._config.get(CONF_SCENE) is not None:
+            if self._config.get(CONF_SCENE) < 20:
+                self._scenes = SCENE_LIST_RGBW_255
+            elif self._config.get(CONF_BRIGHTNESS) is None:
+                self._scenes = SCENE_LIST_RGB_1000
+            else:
+                self._scenes = SCENE_LIST_RGBW_1000
+            for scene in self._scenes:
```

Comment:
Should be fine to just assign `self._effect_list = self._scenes` here

Reply:
`self._scenes` is a `dict` and `self.effect_list` is just the keys, using `self._effect_list = list(self._scenes.keys())` instead